module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -482,9 +482,33 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             $itemResponse = $this->makeRequest('GET', '/item-storage/items', $query);
             $itemBody = json_decode($itemResponse->getBody());
             $notesFormatter = function ($note) {
-                return $note->note ?? '';
+                return !$note->staffOnly && $note->note ? $note->note : '';
             };
+            $textFormatter = function ($supplement) {
+                $format = '%s %s';
+                $supStat = $supplement->statement;
+                $supNote = $supplement->note;
+                $statement = trim(sprintf($format, $supStat, $supNote));
+                return $statement ?? '';
+            };
+            $holdingNotes = array_map($notesFormatter, $holding->notes ?? []);
```

Comment:
I see you added an array_filter to the item notes filtering below; would it make sense to add one here as well for the holding notes?

Reply:
Yes, this is better. Done.